js/src/components/paid-ads/campaign-assets-form.test.js

```diff
@@ -191,7 +191,7 @@ describe( 'CampaignAssetsForm', () => {
 		expect( children ).toHaveBeenLastCalledWith( formContextSchema );
 	} );
 
-	it( 'Should fall back to the custom level for the initial form values when there is no level available', () => {
+	it( 'Should fall back to the current level for the initial form values when there is no level available', () => {
```

Comment:
@ankitrox I feel the label is a bit misleading here because there's no fall back to the current level. The current level is always selected when editing a paid ads campaign.